### PR TITLE
sp_load_solr_lineofservice

### PR DESCRIPTION
**Pull request by carresemata**
Created sp_load_solr_lineofservice in snowflake.
Validation:
* We have the same rows in SQL server and in snowflake
* Data seems to be the same for all columns
* Last column UpdatedSource is different as now we are the ones doing the update (before it was dbo now is our email)

### CODE REVIEW

File: migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql [hunk 1]
```diff
@@ -0,0 +1,138 @@
+-- Show_spuSOLRLineOfServiceGenerateFromMid
+CREATE OR REPLACE PROCEDURE SHOW.SP_LOAD_SOLRLINEOFSERVICE()
```
Comment (omargoldpr) [inline migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql:2]: Add the Database to the procesure name
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_SOLRLINEOFSERVICE() 
```

File: migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        midLine.LineOfServiceTypeCode,
+        midLine.LineOfServiceDescription,
+        midLine.LegacyKey,
+        midLine.LegacyKeyName,
+        CURRENT_TIMESTAMP AS UpdatedDate,
+        CURRENT_USER AS UpdatedSource
+    FROM
+        Mid.LineOfService midLine
+    WHERE
+        midLine.LineOfServiceID IN (
```
Comment (omargoldpr) [inline migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql:54]: Replace the subquery with a CTE and the where with a JOIN

**Review by omargoldpr — CHANGES_REQUESTED** [review]

File: migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql (hunk 1 above)
Comment (carresemata) [reply migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql:2]: Done

File: migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql (hunk 2 above)
Comment (carresemata) [reply migration_original/ODS1Stage/tables/Show/SOLRLineOfService/spu_translated_SOLRLineOfService.sql]: WITH cte_id AS (
            SELECT
                DISTINCT LineOfServiceID
            FROM
                Show.SOLRLineOfServiceDelta
            WHERE
                StartDeltaProcessDate IS NULL
                AND EndDeltaProcessDate IS NULL
                AND SolrDeltaTypeCode = 1 --INSERT/UPDATEs
                AND MidDeltaProcessComplete = 1
)
SELECT
        midLine.LineOfServiceID,
        midLine.LineOfServiceCode,
        midLine.LineOfServiceTypeCode,
        midLine.LineOfServiceDescription,
        midLine.LegacyKey,
        midLine.LegacyKeyName,
        CURRENT_TIMESTAMP AS UpdatedDate,
        CURRENT_USER AS UpdatedSource
    FROM
        Mid.LineOfService midLine
    JOIN
        cte_id ON midLine.LineOfServiceID = cte_id.LineOfServiceID

**Pull request merged**